SUPPORT.md

```diff
@@ -0,0 +1,44 @@
+# Getting support for Julia
+
+As well as the issue tracker here on GitHub, Julia is supported by several other online and offline communities.
+If you have come here to open an issue on the GitHub repository, you might want to want to consider the following options first.
+
+We expect that your participation in any Julia community follows the [Julia Community Standards](https://julialang.org/community/standards/).
+
+## Discussion forums
+
+The [Julia Discourse forum](https://discourse.julialang.org/) is the official and most active discussion venue for the Julia language. The Discourse forum is the right place to do any of the following:
+
+- **Keep up to date** on important Julia announcements like language releases and JuliaCon.
+- **Ask a question** about any aspect of using or developing Julia or its associated packages and tools.
+  The Discourse forum is the best place to get a fast, high-quality answer to your Julia question.
+  There are subcategories for various Julia application domains—for example, machine learning, finance, visualisation or GPUs.
+  Questions from newcomers are always welcome: you can use the "first steps" tag to highlight your question as one that would benefit other new users.
+- **Announce** a new package or solicit feedback about something you have been developing.
+- **Advertise** Julia-related jobs and Julia meetups.
+
+The Discourse forum replaced two mailing lists, [julia-users](https://groups.google.com/forum/#!forum/julia-users) and [julia-dev](https://groups.google.com/forum/#!forum/julia-dev), at the end of 2016.
+You may want to search those lists to see if your question has already been raised.
+
+## Chat rooms
+
+If you have a short one-off question, or want to talk in real-time with other users and developers, or aren't quite sure what kind of question you are asking, then a chat room is a good place to start.
+
+The official Julia chat room is the [invite-only Slack](https://julialang.slack.com/), which is open to members of the community in good standing.
+To request access you should visit https://slackinvite.julialang.org/ to register.
+
+There is an unofficial but active [Gitter room](https://gitter.im/JuliaLang/julia), and many Julia packages and projects have their own Gitter rooms as well.
+There is also a [#julia](http://webchat.freenode.net/?channels=julia) IRC channel on Freenode connected to the Gitter room.
+
+## Other unofficial communities
+
+Stack Overflow is a common destination for short "how-do-I" questions about software development.
+You can ask questions using the [julia-lang](https://stackoverflow.com/questions/tagged/julia-lang) tag on Stack Overflow, or the [julia](http://es.stackoverflow.com/questions/tagged/julia) tag on Stack Overflow en Español.
+
+On Twitter, use the [#julialang](https://twitter.com/search?q=%23julialang) hashtag.
+
+There is a small [Julia subreddit](https://www.reddit.com/r/Julia/) for casual discussion.
+
```

Comment:
I think it is worth mentioning our YouTube channel, which has all the JuliaCon videos.